### ISSUE TITLE
"Ticket Id: 231866954 Number: 90 Preparation of Pitch Mockups"

### ISSUE THREAD

**Issue by yashsmartnoter** [new-smartnoter/assembla_sn-future#2]
"Hi @anilch!, \n\nHere is a draft narrative for the portion of the pitch that will include mock-up images that you, me, and Surya will quickly create together. Each paragraph will be spoken by me when giving the pitch, and there will be a slide for each mockup. I will be providing sketches of the mockups.\n\nPortion of Pitch Narrative for Mock-ups\n\nThere will be other parts of the pitch first. Then, these slides will be displayed.\n\nSuppose you are a science professor preparing a general introductory biology course. Your dashboard makes it easy to create and edit an exceptional course that seamlessly integrates convergent media into the assignments. \n\nYou may choose to start with your videotaped lecture, which you will then easily divide up into Virtual Segments, based on each of the topics covered. You can enhance each topic with a QueryCaps question that will appear below the main screen, that alert the students to what they should take away from that part of the lecture. \n\nYou’ll probably want to create QuikQuizzes that make it easy for students to self-test themselves, and to go back and review a segment they may have missed or forgotten, either with instant-replay of that segment, or later, when they are studying for an exam. \n\nBut that’s just the beginning. You can relate each Virtual segment to real-life issues by creating Relevance Networks. Put simply, you can link any topic to a rich collection of relevant information, from any source, including libraries that you share with your professional colleagues, or any type of streaming media, interactive website, or static image, from anywhere on the Internet. \n\nFor example, you might encourage students to choose from among a set of five different Internet resources, and then write, in their own words, a brief summary of what they have learned. \n\nYour online dashboard allows you, or your Teaching Assistants, who might be located nearby, or hundreds of miles away, to easily manage student enrollment and schedule an entire syllabus of assignments, and you can flexibly integrate any other website that can be displayed on a browser and use that for any purpose, such as secure midterms or finals. Your class is not constrained by the limitations of any current website or software that you use for your class--you and your students can seamlessly switch from one application to another. \n\nNow suppose you are a biology student enrolled in this class. On your desktop, you will take full advantage of the dual-browsing capabilities. On one side of the screen you can watch the main lecture segmented with QueryCaps, and have full access to the web of relevant links, while on the other side of the screen, you can display any web-based or local application in a native browser. \n\nOr suppose you are a graduate student getting a masters degree in Journalism. You will be able to prepare a thesis project that seamlessly integrates excerpts or examples from a variety of news sources, or if you are focused on the history of photojournalism, you can easily display the actual photographs or video clips that you have referenced and analyzed in your thesis. \n\nThen, there will be some other kind of slide for this conclusion,  and the other parts of the pitch will be given. \nThe truth is, ViewBacca is so flexible that the course design and actual online experience are limited only by the creativity of the professors and the students. \n"

**Comment by yashsmartnoter** [comment]
Hi @genelevinson!

Thank you for sharing the version 3 of VBCMHE. I will go through in detail and come up with Questions if any before our call on Monday.

Thanks,
Anil.C

**Comment by yashsmartnoter** [comment]
Hi @anilch!
I've shared a new version 3 document with your email. Please review and share and discuss with the team. Enjoy the rest of the weekend. 
Kind regards, 
Gene
https://docs.google.com/document/d/1NEkc3GOwPMh1eqVLaSjN_Rf4VC1MH11S9v5TJ2ItweU/edit?usp=sharing

**Comment by yashsmartnoter** [comment]
Hi @anilch!
No rush to look at this-- can certainly wait until your morning. Just wanted to make it available now. Here is the document I mentioned-- it has the document you shared with me at the top, plus new notes about Slide 1, and a sketch, at the top, which we can discuss  during or after your Morning. Ping me on Signal if you want to chat about it in your early morning (say before 10 pm my time) or in my morning tomorrow. Feel free to make a copy of this read only Google Drive file (shared with your email address)  and share with the team (and me) in a new ticket in the regular team space. 
Thanks, 
Kind regards, 
Gene
https://docs.google.com/document/d/1FGuShfLEpxm9ucRnangohLJaNw8xDXRSqrcQeCPtjOQ/edit?usp=sharing

**Comment by yashsmartnoter** [comment]
Hi @anilch!
Yes, good, I agree that more brainstorming and discussion of how to simplify this for the users is important.

**Comment by yashsmartnoter** [comment]
Hi @genelevinson!

Agree, there are a lot of things to cover to fulfill this requirement and this can only be done using widgets. But I think it's not that straight forward and it should also be ease of use to the users.

The document which you have shared is understandable.

Once we had an initial discussion along with Surya on the group call, Surya will imagine and comeup with an initial sketch for Slide 1 as per the document.


Thanks,
Anil.C

**Comment by yashsmartnoter** [comment]
Hi @anilch!
Here is a link to a shared Google document in progress that I shared with your email address. https://docs.google.com/document/d/1QXJVYhv3KL5VZuzbm3eLR3XM6skHcVFwN9_lNBCHUjY/edit?usp=sharing

Unfortunately there are no shortcuts to the sketches--this approach, using Widgets, will be essential for both the Pitch Deck and Demo to succeed, and these steps need to be done before actually sketching what the Widgets look like. This is a work in progress and is not complete, and I will work on this again, hopefully before our meeting. But there is a lot to discuss here as it is. 
Kind regards, 
Gene

**Comment by yashsmartnoter** [comment]
Hi @genelevinson!

Thanks for sharing the details.

Yes, I will also review it before our call and also will share it with Surya now.


Thanks,
Anil.C

**Comment by yashsmartnoter** [comment]
Hi @anilch!
Here are draft definitions and descriptions that we will be using. Please share with with Surya also, at this time. Thanks. 
Kind regards, 
Gene

DEFINITIONS:
COURSE
A complete unit of instruction in a particular subject area. College courses typically are published in a catalog, and meet certain requirements for graduation. Courses have a title, a number, and a description in the catalog. Students enroll in courses, turn in assignments, are graded on their performance, and must complete the course with a satisfactory grade to use towards graduation in a particular major. 

Example: 
Title: General Biology for Non-Majors 
Number: Biology 101
Description: A comprehensive overview of modern biology topics for non-majors, including molecular, cellular, and developmental biology, ecology and biotic and non-biotic factors, microbiology and pathogens, evolution and natural history, and anatomy and physiology of animals, plants, and fungi. 

SYLLABUS
Detailed description of a COURSE, which usually includes a detailed outline of topics that are covered, usually in the sequential order in which they are presented. 

CLASS
One scheduled instance of a COURSE, which typically takes place during a semester or quarter, and has a list of students who are officially enrolled and will receive a number of units of course credit as well as a grade when the course is completed. 

CLASS SCHEDULE
For on-campus, physical courses, the class schedule includes a complete sequence of dates, times, and physical locations where the COURSE LESSONS, LECTURES, LABS, and/or GROUP ACTIVITIES will take place, as well as scheduled EXAMINATIONS such as QUIZZES, MIDTERMS or FINALS. 

STUDENT ASSIGNMENTS
A student deliverable, which may include attendance of specific lectures, activities or events, documents or other materials that must be submitted to the professor or teaching assistants, participation in group projects, examinations of various kinds, projects or papers, and other activities that are part of the course syllabus. 

LESSON PLAN
A detailed plan for a scheduled class meeting (in-person, streaming, or on-demand), which includes both professors’ and students’ presentations, activities, and deliverables, which may include specific online meetings, assignments, lectures, examinations, or other activities. Traditionally scheduled for a specific date, time and location for in-person meetings; may be more personalized and flexible for online and on-demand participation. 

CENTRAL TITLE
A CENTRAL TITLE represents the main thread of an LESSON PLAN that the student will attend on-demand. One or more CENTRAL TITLES will be displayed consecutively, end-to end, in a typical on-demand session. CENTRAL TITLES are often represented by digital video as the media type, but they may be represented by other media types--usually types that change over time, such as audio presentations or slide shows. 

CENTRAL TITLE RUNNING LENGTH
Each CENTRAL TITLE will have a beginning and an end, and the time that elapses between these represents its RUNNING LENGTH.  

CENTRAL MEDIA STREAM (CMS)
Each LESSON PLAN has one or more CENTRAL TITLES. A typical CMS might include a  CENTRAL TITLE that consists of a video lecture that was prepared by the professor.

CMS OR LESSON RUNNING LENGTH
The CMS OR LESSON RUNNING LENGTH consists of the sum of the length of one or more CENTRAL TITLE RUNNING LENGTHS. 

CONVERGENT MEDIA LESSON STREAM (CMLS)
A CMLS is typically defined in detail by a specific LESSON PLAN. It has a LESSON RUNNING LENGTH defined by the interval between its beginning and ending timestamps. It has one CENTRAL MEDIA STREAM of one or more CENTRAL TITLES. that are typically video, audio, or static slide presentations, such as lectures. 

CENTRAL TITLE VIRTUAL SEGMENTS 
Each CENTRAL TITLE is typically divided into VIRTUAL SEGMENTS by the professor and/or teaching assistants. Each VIRTUAL SEGMENT has a beginning and an ending time stamp, and the SEGMENT RUNNING LENGTH is defined by the difference between those time stamps. Each VIRTUAL SEGMENT represents a topic (defined by the professor and/or teaching assistants) that is presented within the sequence of topics that are part of  a 
CENTRAL TITLE. This topic provides a clear point of reference that captures the main subject matter, and is typically attached to QUERY CAPS, QUIKQUIZZES, and RELEVANCE NETWORKS. 

QUERYCAPS
QUERYCAPS are typically questions--or other text--that define the topic that will be covered in a CENTRAL TITLE VIRTUAL SEGMENT. They are displayed below the CENTRAL TITLE screen from the beginning to the end of a VIRTUAL SEGMENT. They are designed to engage the student, encourage critical thinking, and define the learning objective of a VIRTUAL SEGMENT. 

QUIKQUIZZES
Quikquizzes are optional multiple choice answers from which the student can select, and represent a fun, informal way for students to test their knowledge as they proceed through a lesson. If they get the answer wrong, they have opportunities to replay the VIRTUAL SEGMENT, to take the quiz again, to study other parts of the RELEVANCE NETWORK of that VIRTUAL SEGMENT, and to find the correct answer. Statistics on individual student, class, or general class performance can provide valuable feedback to the professor and teaching assistants on the effectiveness of that VIRTUAL SEGMENT of a lesson plan.  

RELEVANCE NETWORK OF A VIRTUAL SEGMENT
A RELEVANCE NETWORK consists of PRIMARY AND SECONDARY TITLES that are relevant to the topic of a CENTRAL TITLE VIRTUAL SEGMENT. They are attached to specific VIRTUAL SEGMENTS OF A CENTRAL TITLE. They typically include a broad range of different media types, including titles (or websites) that can be INTERNAL to the college or organization or EXTERNAL, i.e. from other sources on the Internet. 

PRIMARY TITLES
Primary titles are titles, from a variety of different sources and media types, either internal to the college or organization or external from the Internet, that have been selected by professors or teaching assistants or students, that are relevant to the topic in a CENTRAL TITLE VIRTUAL SEGMENT. During the duration of the CENTRAL TITLE VIRTUAL SEGMENT, they are displayed as thumbnails that students can choose to click on. Guidance from the professor or teaching assistants will inform students of the best practices for viewing these PRIMARY TITLES, and whether or not there are assigned deliverables that the student is expected to turn in, according to a schedule. When the student clicks on a PRIMARY TITLE thumbnail, this will pause the CENTRAL TITLE and display the PRIMARY TITLE presentation on the screen. If they are video, audio, or slide show presentations, they typically have a beginning and an end which define a PRIMARY TITLE RUNNING LENGTH. Or they may be websites or static media such as images or text. 

PRIMARY TITLE VIRTUAL SEGMENTS
If a PRIMARY TITLE has a RUNNING LENGTH, then it can be divided into PRIMARY TITLE VIRTUAL SEGMENTS (PTVS) and these PTVS can have SECONDARY TITLES and QUERYCAPS and QUIKQUIZZES attached.

SECONDARY TITLES
Secondary titles are attached to PRIMARY TITLE VIRTUAL SEGMENTS, according to their relevance, and represent a second level of relevant information available to the student. The professor will provide guidance regarding best practices and deliverables associated with these SECONDARY TITLES. 

SECONDARY TITLE VIRTUAL SEGMENTS
If a SECONDARY TITLE has a RUNNING LENGTH, then it can be divided into PRIMARY TITLE VIRTUAL SEGMENTS (PTVS) and these PTVS can have QUERYCAPS and QUIKQUIZZES attached.

PROFESSORS’ AND ASSISTANTS’ MEDIA ASSETS
The professor’s or teaching assistant’s dashboards will be populated with MEDIA ASSETS, which represent either internal or external titles and links that can be utilized when preparing LESSON PLANS which include CENTRAL TITLES and RELEVANCE NETWORKS OF PRIMARY AND SECONDARY TITLES. 

VIEWBACCA ON-DEMAND LESSON (VODL)
The VODL is the name given to the complete on-demand experience that is typically based on a LESSON PLAN. This is typically available on-demand, but will be assigned according to a schedule, and the student is accountable for following that schedule. Materials available to each student include the CONVERGENT MEDIA LESSON STREAM (CENTRAL TITLES, PRIMARY TITLES, and SECONDARY TITLES, as well as ASSIGNMENTS (deliverables), guidance from the professor and teaching assistants, as well as access for approved ways to contact the professor, teaching assistants, and other members of the class, which form an approved CLASS SOCIAL NETWORK. Each student will interact with and personalize this experience, according to the rules and guidance provided by the professor and by their peers.

**Comment by yashsmartnoter** [comment]
Thanks @anilch!
Yes, that plan makes sense.
Let me explain the details of what I meant by the following lines: 

"Any sketches depend on the assumptions and features--the overall approach and specific design choices that we make-- for our UX.
For that reason, the chosen UX framework is a dependency that must be in place--at least conceptually, at a high level-- before sketches for mockups can be made."

Specifically, what I meant (but did not fully explain) concerns the professor's dashboard approach, and sketch 1 in particular (and others as well) depend on that approach. 

The professor's dashboard approach needs to seamlessly integrate creation tools and editing tools, with a simple learning curve, and obvious user-friendly approach, for these functionalities: 
COURSE
SYLLABUS
CLASS
CLASS SCHEDULE
STUDENT ASSIGNMENTS
LESSON
MAIN SEGMENTED SCREEN
PRIMARY AND SECONDARY LINKS
QUERY CAPS AND QUIK QUIZ

This is what I am going to explain in more detail in a document that I am preparing today. Specifically, the ways that these functionalities are represented on the professor's dashboard--with proper flexible options for how they are edited and displayed-- with several redundant ways of displaying--is what needs to be done right. 
This will become more clear when I share the document I am working on now. The display choices are challenging, but critical. 
Thanks, 
Kind regards, 
Gene
